scripts/setup-react-native.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
did you write this / need to credit someone? and also need to fix up parts of it